locked-asset/simple-lock-energy/src/penalty.rs

```diff
@@ -0,0 +1,80 @@
+use common_structs::Epoch;
+
+use crate::lock_options::EPOCHS_PER_YEAR;
+
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+#[derive(TypeAbi, TopEncode, TopDecode)]
+pub struct PenaltyPercentage {
+    pub first_threshold: u16,
+    pub second_threshold: u16,
+    pub third_threshold: u16,
+}
+
+#[elrond_wasm::module]
+pub trait LocalPenaltyModule {
+    fn calculate_penalty_percentage_full_unlock(
+        &self,
+        lock_epochs_remaining: Epoch,
+        penalty_percentage_struct: &PenaltyPercentage,
+    ) -> u64 {
+        let first_threshold_penalty = penalty_percentage_struct.first_threshold as u64;
```

Comment:
Why not have the struct's variables as u64 for consistency? This will also help to not convert the values each time, as most variables of this type throughout the DEX contracts are of type u64.

Reply:
+1